tests/components/fujitsu_hvac/test_init.py

```diff
@@ -6,56 +6,56 @@
 from freezegun.api import FrozenDateTimeFactory
 import pytest
 
-from homeassistant.components.fujitsu_hvac.const import API_REFRESH_SECONDS
-from homeassistant.const import STATE_UNAVAILABLE, STATE_UNKNOWN
+from homeassistant.components.fujitsu_hvac.const import API_REFRESH, DOMAIN
+from homeassistant.const import STATE_UNAVAILABLE, Platform
 from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
 
 from . import setup_integration
 
 from tests.common import MockConfigEntry, async_fire_time_changed
 
 
-async def test_coordinator_sign_in_exception_get_devices(
+async def test_auth_failure(
     hass: HomeAssistant,
     freezer: FrozenDateTimeFactory,
-    mock_devices: list[AsyncMock],
     mock_ayla_api: AsyncMock,
     mock_config_entry: MockConfigEntry,
 ) -> None:
-    """Test that the entities become unavailable if we fail to update them because of an auth error while getting the devices."""
+    """Test entities become unavailable after auth failure."""
     await setup_integration(hass, mock_config_entry)
 
     mock_ayla_api.async_get_devices.side_effect = AylaAuthError
-    freezer.tick(API_REFRESH_SECONDS)
+    freezer.tick(API_REFRESH)
     async_fire_time_changed(hass)
     await hass.async_block_till_done()
 
-    assert all(e.state == STATE_UNAVAILABLE for e in hass.states.async_all())
+    assert hass.states.get("climate.testserial123").state == STATE_UNAVAILABLE
+    assert hass.states.get("climate.testserial345").state == STATE_UNAVAILABLE
```

Comment:
This seems brittle to me. why hardcode the entity id when you can use what I had previously which would still work even if the serial was changed in the fixture ?

Reply:
The other setup also worked, but it left me questioning, does only 1 have this problem, or do they both? And even though I love a good oneliner, I also like to know what is actually going wrong